build.sbt

```diff
@@ -142,7 +143,15 @@ val ignoredABIProblems = {
   import com.typesafe.tools.mima.core._
   import com.typesafe.tools.mima.core.ProblemFilters._
   Seq(
-    exclude[MissingTypesProblem]("com.twitter.chill.storm.BlizzardKryoFactory")
+    exclude[MissingTypesProblem]("com.twitter.chill.storm.BlizzardKryoFactory"),
+    exclude[IncompatibleResultTypeProblem]("com.twitter.chill.ClosureCleaner.*"),
```

Comment:
how inconvenient do you think it would be to make this binary compatible? Binary incompatibility is a pain if we could use `abstract class` to fix?